### ISSUE TITLE
[Request] Dark version

### ISSUE THREAD

**Issue by somniumAeternam** [keeferrourke/la-capitaine-icon-theme#4]
Hi! Very nice theme. Would it be possible, that you also provide a dark version of this icon theme?

**Comment by keeferrourke** [comment]
When you suggest making a dark version of the icon theme, what exactly do you mean? Do you want the icons to have increased contrast against dark backgrounds? 

Depending on your GTK theme, using the dark mode setting, the icons should still appear clearly; see the screenshot of my Nautilus window below.

![screenshot from 2016-07-16 14-01-56](https://cloud.githubusercontent.com/assets/6888381/16896311/e06df198-4b5d-11e6-9b4a-1c69cff8c624.png)

I'm willing to make changes as you request them, I just need a little more to go off from :smiley:

**Comment by somniumAeternam** [comment]
I'm using the Plasma 5.7 Desktop from KDE and when I use a dark theme (color scheme), some icons are dark grey on dark grey background, they should be light grey.

Here is an example:

![spectacle sl7461](https://cloud.githubusercontent.com/assets/9334916/16909319/4a673814-4cd2-11e6-83ce-e509d62f762d.png)

**Comment by keeferrourke** [comment]
Problem noted &ndash; I'll add this to my to-do list.

**Comment by DarkknightAK** [comment]
Hi,

I'm one of the KDE Breeze icon developers and I already did the recolor stuff in Breeze-icons so if you like I'd like to help.

In KDE it's possible that the icon theme use the color from the color scheme, very cool something like symbolic stuff in Gnome.

what do you think?

**Comment by keeferrourke** [comment]
Hi @DarkknightAK thanks for your help! I very much appreciate it, however please review commit 274ec9e9181120f2563c47b04015af542874a076, as I cannot merge this.

**Comment by DarkknightAK** [comment]
styling in kde work now.

**Comment by keeferrourke** [comment]
Thanks to @DarkknightAK this issue should be partially fixed; at least in KDE, the icon theme will automatically be dark or light depending on the set theme. Work still needs to be done in GNOME/GTK+ desktop environments.

**Comment by DarkknightAK** [comment]
can you update the kde store with the styled icons? I'd like to make a blog post and link to your La Capitain icons in the kde.store

**Comment by keeferrourke** [comment]
@DarkknightAK I can make another point release, but I'd prefer to hold off for a bit until a few more of the smaller issues are fixed. Ping me again before Christmas.

**Comment by kajzersoze** [comment]
Too bad there is no dark version, hence I can't use this icon set, however I don't see why you can't do that with simple script and Inkscape, there's no need to convert them manually.

**Comment by keeferrourke** [comment]
@kajzersoze feel free to do so yourself. I'm aware that it's a simple fix, however I don't want to maintain two versions of the icon set at this time. I'll write some scripts that will take care of the bulk of this work eventually... I've just been very busy as of late :)

**Comment by RaitaroH** [comment]
@keeferrourke Hi there. I actually made a dark variant of this. Is not perfect but you should be able to do it better.
I used the script 1 from: https://github.com/erikdubois/Arc-Theme-Colora
After I installed the necessary software in script 0 and gpick I went to work. I replaced the colors such as 343434 or 555555 or 4d4d4d with f2f2f2 (this is what breeze dark uses)
This is how it looks like:
Original: https://i.imgur.com/wazcJFD.png
Modified: https://i.imgur.com/qlLQ36q.png
I also added a archive for the dark variant (and that script) here: 
https://github.com/RaitaroH/Xfce-Goodies-Tutorial/blob/master/la-capitaine-dark.zip
I didn't make another github page just for this. Download the zip yourself and maybe use it it the new version. This really shouldn't take you more than on hour to make at worst. I do want to ask you to make a complete dark theme, that is with all the app icons and such instead of inheriting the original. Same as I did. (I don't want to inherit too many icons sets)

**Comment by keeferrourke** [comment]
@RaitaroH hey there! thanks for recolouring the icons for me, that's going to save me a lot of work. When I get a chance I'll incorporate that archive into La Capitaine and have the light/dark variants ready for a 0.5.0 release.

>  I would like to have the dark variant as a complete theme the same way Breeze Dark is but is your call (you could copy the files around during the installation)

I was planning on doing this is by including `*-dark` folders in the main La Capitaine project (and respectively renaming the existing folders to `*-light`), which would then get symlinked to the proper names when the configuration script is run. This would also keep the size of the project to a minimum, as `apps/scalable` and other directories would not need to be duplicated. If your issue is that you don't want to run a config script, I can attach dark/light gzipped tar files to each release on Github.

Does this make sense?

> and hey maybe give me and Erik a shoutout :)

Definitely! Remind me who Erik is? Whoever helped with this recolouring is getting a nice comfy spot in [Thanks.md](https://github.com/keeferrourke/la-capitaine-icon-theme/blob/master/Thanks.md), credits in the commit message, and a shoutout on my website :grin:

As for a timeline for this... You can *probably* expect this update to happen sometime this week. If not then next week.

> So basically create a link of empathy.svg or emesene.svg (better) with the name cs-input-method.svg. Also can you make the icons fit more with one another?

Matter of opinion, but is a chat program's icon really a great fit for input method settings? I think I can probably do better than that. I'll look into designing an icon for it. Secondly, I think the icons in La Capitaine *do* fit well with one another. I'm not a huge fan of your mostly blue set up, so I probably won't change them in the way you expect. Do however keep in mind that I'm constantly redesigning icons where I see fit, and the settings panel is somewhere I want to focus a lot of attention.

> Also you will need to add nemo-eject.svg to actions (this is the eject icon for devices. Add a shortcut to media-eject.svg (action 22x22) with the name nemo-eject.svg and the problem is solved as well.

Sure thing!

> I would also recommend to make a user-trash-full and user-trash a bit more different. I like how in Breeze the user-trash-full becomes red so you really notice is full.

Agree to disagree here. I think it's fairly obvious when there's trash and when there's not, in both the symbolic and scalable versions of the icons. You can whatever you want with your local version though :wink:

**TL;DR**
To do on this issue:

- [ ] incorporate dark zip archive into main project
- [ ] update configuration script to toggle between light/dark themes
- [ ] add `nemo-eject.svg` to `actions/22x22`

**Comment by RaitaroH** [comment]
@keeferrourke Thanks for the quick response.  Regarding the settings icons and the trash icons, those were suggestions. In order to keep the icon set untouched I just have another one with the changes I like and then inherit this one :)
Also I cannot be sure if all dark icons are fine. They looked fine to me and I am using them right now and I don't see any issues. It is possible that some icons didn't recolor well though, so you should give them a nice look before you incorporate them.

Thank you for your dedication and the awesome icon set.

**Comment by keeferrourke** [comment]
@RaitaroH noted, I'm slowly getting through a lot of redesigns, so not worry &mdash; I will address this eventually.

**Comment by keeferrourke** [comment]
Alrighty. Looks like I'm done for now. Thanks again to @RaitaroH for recolouring so many icons!

For those of you that want the dark version, follow these simple instructions:

 * pull the latest commits from master
 * run `./configure` and answer the prompts according the set up you want; by default La Capitaine still uses the light variant of the theme.

**Comment by RaitaroH** [comment]
Download it, move it to .icons, configured it, and works well. I think you can close this thread now,

**Comment by kajzersoze** [comment]
Seems fine, good job all !
